packages/plugins/users-permissions/admin/src/pages/AdvancedSettings/index.js

```diff
@@ -212,4 +219,4 @@ const AdvancedSettingsPage = () => {
   );
 };
 
-export default AdvancedSettingsPage;
+export default Protected;
```

Comment:
Can you name these pages "ProtectedXXX" where XXX is the previously exported component?